src/sentry/integrations/slack/notify_action.py

```diff
@@ -121,13 +131,18 @@ def after(self, event, state):
 
         def send_notification(event, futures):
             rules = [f.rule for f in futures]
-            attachment = build_group_attachment(event.group, event=event, tags=tags, rules=rules)
+            attachments = [build_group_attachment(event.group, event=event, tags=tags, rules=rules)]
+            # check if we should have the upgrade notice attachment
+            if get_integration_type(integration) == "workspace_app":
+                if random.random() < UPGRADE_MESSAGE_FREQUENCY:
```

Comment:
anyway we could log some metrics/logging here? I would really love to be able to see counts of how many times we are actually sending this, and also if possible, by org. I think this would be helpful in getting a sense of how much we should be increasing the %

Reply:
@MeredithAnya Great idea. We can probably put this to datadog.